exporter/exportertest/nop_exporter.go

```diff
@@ -19,30 +19,28 @@ import (
 
 	"github.com/open-telemetry/opentelemetry-collector/component"
 	"github.com/open-telemetry/opentelemetry-collector/consumer/consumerdata"
+	"github.com/open-telemetry/opentelemetry-collector/internal/data"
 )
 
-// NopExporterOption represents options that can be applied to a NopExporter.
-type NopExporterOption func(*nopExporter)
-
-type nopExporter struct {
+type nopExporterOld struct {
 	name     string
 	retError error
 }
 
-func (ne *nopExporter) Start(host component.Host) error {
+func (ne *nopExporterOld) Start(_ component.Host) error {
```

Comment:
this is a good pattern, `_` for unused, do you know if any of the tools can enforce it?

Reply:
I don't know about tools but goland helps with this.